src/Evolution/Systems/CurvedScalarWave/BoundaryConditions/ConstraintPreservingSphericalRadiation.hpp

```diff
@@ -0,0 +1,155 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+#include <optional>
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/BoundaryConditions/Type.hpp"
+#include "Evolution/Systems/CurvedScalarWave/BoundaryConditions/BoundaryCondition.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace domain::Tags {
+template <size_t Dim, typename Frame>
+struct Coordinates;
+}  // namespace domain::Tags
+/// \endcond
+
+namespace CurvedScalarWave::BoundaryConditions {
+
+/*!
+ * \brief Implements constraint-preserving boundary conditions with a second
+ * order Bayliss-Turkel radiation boundary condition.
+ *
+ * \details The Bayliss-Turkel boundary conditions are technically only valid in
+ * flat space and should therefore only be used at boundaries where the
+ * background spacetime is approximately Minkwoski such as (sufficiently far
+ * out) outer boundaries for asymptotically flat spacetimes. Small reflections
+ * are still likely to occur.
+ *
+ * The constraint-preserving part of the boundary conditions are set on the time
+ * derivatives of the evolved fields according to \cite Holst2004wt . The
+ * physical Bayliss-Turkel boundary conditions are additionally set onto the
+ * time derivative of \f$\Pi\f$.
+ *
+ * The constraints are defined as follows:
+ *
+ * \f{align*}{\mathcal{C}_i&=\partial_i\Psi - \Phi_i=0, \\
+ *  \mathcal{C}_{ij}&=\partial_{[i}\Phi_{j]}=0 \f}
+ *
+ * Equations (40) and (41) from \cite Holst2004wt give constraint-preserving
+ * boundary conditions on the characteristic fields which we convert back to
+ * conditions on the evolved fields:
+ *
+ * \f{align*}{\partial_{t} \Psi&\to\partial_{t}\Psi + \lambda_\Psi n^i
```

Comment:
I still don't quite understand how this follows from Eq 40, e.g. there's a gamma_1 in lambda_psi here but Eq 40 has no gamma_1. What am I missing?